pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -471,20 +471,9 @@ self: super: {
     dependencies = with self; [ lush-nvim ];
   });
 
-  himalaya-vim = buildVimPluginFrom2Nix {
-    pname = "himalaya-vim";
-    inherit (himalaya) src version;
+  himalaya-vim = super.himalaya-vim.overrideAttrs (old: {
     dependencies = with self; [ himalaya ];
-    configurePhase = ''
-      cd vim
-      substituteInPlace plugin/himalaya.vim \
-        --replace 'if !executable("himalaya")' 'if v:false'
-    '';
-    postFixup = ''
-      mkdir -p $out/bin
-      ln -s ${himalaya}/bin/himalaya $out/bin/himalaya
-    '';
-  };
```

Comment:
any reasons why those were removed?

Reply:
Whatever the rewrite was when the repository migrated to the open SourceHut, these were no longer required. I just rebuilt my Vim config to double check and it appears to work for me without the lines still.

Reply:
I would bet I didn't have issues because I have the base Himalaya installed